prediction-applier-extension/pom.xml

```diff
@@ -116,6 +116,26 @@
                 </configuration>
             </plugin>
 
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.5.3</version>
```

Comment:
Maybe the plugin version could be specified in parent POM's `pluginManagement` section?